### PR TITLE
Bioconductor package Repitools and its dependencies

### PR DESCRIPTION
**Pull request by iwohlers**
* [x] I have read the guidelines above.
* [x] This PR adds a new recipe.
* [x] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

### CODE REVIEW

File: recipes/r-aroma.affymetrix/bld.bat [hunk 1]
```diff
@@ -0,0 +1,8 @@
+"%R%" CMD INSTALL --build .
+if errorlevel 1 exit 1
```
Comment (bgruening) [inline recipes/r-aroma.affymetrix/bld.bat]: @iwohlers can you remove all bat files from this PR. We do not support Windows in bioconda.

File: recipes/r-aroma.affymetrix/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+  name: r-aroma.affymetrix
+  # Note that conda versions cannot contain -, so any -'s in the version have
+  # been replaced with _'s.
+  version: "3.0.0"
+
+source:
+  fn: aroma.affymetrix_3.0.0.tar.gz
+  url:
+    - http://cran.r-project.org/src/contrib/aroma.affymetrix_3.0.0.tar.gz
+    - http://cran.r-project.org/src/contrib/Archive/aroma.affymetrix/aroma.affymetrix_3.0.0.tar.gz
```
Comment (bgruening) [inline recipes/r-aroma.affymetrix/meta.yaml]: Adding a checksum and removing all unneeded comments would also be great.

File: recipes/r-globals/meta.yaml [hunk 3]
```diff
@@ -31,12 +34,12 @@ build:
...
   build:
-    - r
-    - r-codetools # [not win]
+    - r-base
+    - r-codetools
 
   run:
-    - r
-    - r-codetools # [not win]
+    - r-base
```
Comment (bgruening) [inline recipes/r-globals/meta.yaml]: Please make sure you use r and not r-base.

**Comment by bgruening** [comment]
Thanks @iwohlers - what a giant PR! :)

File: recipes/r-aroma.affymetrix/meta.yaml (hunk 2 above)
Comment (iwohlers) [inline recipes/r-aroma.affymetrix/meta.yaml]: I just used the provided scripts for generating R and Bioconductor packages. How can I add checksums? How can I know which comments are unneeded?

Comment (bgruening) [inline recipes/r-aroma.affymetrix/meta.yaml]: I think you need to do it manually until we have fixed the skeleton generator.

**Comment by iwohlers** [comment]
@bgruening I tried to incorporate all your comments. OK to merge?

**Comment by bgruening** [comment]
More than ok! This is great! Thanks for your contribution!

**Pull request merged**